src/modules/quick_resolution/SettingsLibrary/Resolution.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <vector>
+#include <windows.h>
+#include <winuser.h>
+
+struct Resolution
+{
+    int width;
+    int height;
+
+    //Resolution() {}
+    //Resolution(int _width, int _height)
+    //{
+    //    width = _width;
+    //    height = _height;
+    //}
+};
+
+struct MonitorResolutionSettings
+{
+    LPWSTR displayAdapterName;
+    LPWSTR monitorName;
+    Resolution currentResolution;
+    Resolution* resolutionOptions;
+    Resolution res1; // TODO this should be an array, but it requires more complex marshalling. 
```

Comment:
Not a fan.  Worth the effort if you have time to not hard code # of monitors.  Not a show stopper

Reply:
I can't tell you how shameful it was to write that.. I'll replace it if I can get the marshaling working.